execCommand.html

```diff
@@ -602,10 +602,10 @@ <h3 id="enabled-commands">Enabled commands</h3>
             "command">commands</a> defined here are <a href=
             "#enabled">enabled</a> if the <a href="#active-range">active
             range</a> is not null, its [=range/start node=] is either <a href=
-            "#editable">editable</a> or an <a href="#editing-host">editing
+            "#editable">editable</a> or an <a data-cite="html">editing
```

Comment:
You can move the `data-cite="html"` up to the `<body>`, then you only need to mention it once.

Reply:
Also, editing is just a concept, so actually you can do: [=editing host=] and [=editing hosts=] and get rid of the `<a>` everywhere.